.github/workflows/ci.yml

```diff
@@ -131,7 +131,12 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - uses: actions-rs/cargo@v1
         with:
-          command: test
+          command: install
+          args: nextest
```

Comment:
```suggestion
          args: cargo-nextest
```

Reply:
`cargo` works this way: a binary crate named `cargo-foobar` installed in `~/.cargo` using `cargo install cargo-foobar` could be then called using `cargo foobar`